src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            while (true) {
+                String line = br.readLine();
+                if (line == null) {
+                    break;
+                } else {
+                    String[] splitedLine = line.split(",");
+                    switch (splitedLine[0]) {
+                        case "buy" -> buy += Integer.parseInt(splitedLine[1]);
+                        case "supply" -> supply += Integer.parseInt(splitedLine[1]);
+                        default -> System.out.println("unknown line: " + line);
+                    }
+                }
+            }
+            writeToFile(toFileName, buy, supply);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Catching exceptions and using `e.printStackTrace()` is not recommended. According to the checklist, you should re-throw a `RuntimeException` with a message and the original exception.